expyfun/_externals/h5io/_h5io.py

```diff
@@ -0,0 +1,297 @@
+# -*- coding: utf-8 -*-
```

Comment:
hmm, I copied `mne-python` on this, though I recall alex insisting on having the separate directory.  So I should ditch it?